libraries/Microsoft.Bot.Builder.Dialogs.Debugging/DebugAdapter.cs

```diff
@@ -13,13 +13,13 @@
 
 namespace Microsoft.Bot.Builder.Dialogs.Debugging
 {
-    public sealed class DebugAdapter : DebugTransport, IMiddleware, DebugSupport.IDebugger
+    public sealed class DebugAdapter : DebugTransport, IMiddleware, IDialogDebugger
```

Comment:
This is not an adapter like BotFrameworkAdapter, judging by the interfaces it implements. In our world, adapter has a special meaning. Should we consider another name for this abstraction? DialogDebugger or similar sounds better for something that has a StepAsync method.